ssh/tailssh/incubator.go

```diff
@@ -1089,6 +1089,10 @@ func (ia *incubatorArgs) loginArgs(loginCmdPath string) []string {
 
 func shellArgs(isShell bool, cmd string) []string {
 	if isShell {
+		if runtime.GOOS == "freebsd" {
```

Comment:
Impact of this is that we won't be sourcing `/etc/profile` or `~/.profile` (since we won't be acting as a login shell) on freebsd, correct? 

Do you have a feel for whether not acting as a login shell and reading in these files will be odd / different enough UX from other SSH implementations that we should hold off on unblocking this until we have something to provide or emulate `-l` behaviour on freebsd?

Reply:
Correct, without a login shell, we won't load the user's profile. There are also aspects of PAM integration that we miss out on. I looked for alternatives like `exec -l`, but exec doesn't support `-l` on FreeBSD either. Possibly the `login` command would work, but that potentially gets us into bigger behavior changes.

Reply:
Actually, as long as you're running `tailscaled` as `root`, it already uses `login`, so the problem at hand only applies if you're not running `tailscaled` as `root`.

Reply:
That all makes sense to me! Wonder if it is worth having our own logic to emulate login at some point, think this is what openssh does iirc. Definitely a can of worms going down that path though I'm sure, and definitely not something to look at in the short term for this issue.